flixel/text/FlxBitmapText.hx

```diff
@@ -1208,32 +1209,40 @@ class FlxBitmapText extends FlxSprite
 
 		var tabWidth:Int = spaceWidth * numSpacesInTab;
 
+		var charUt8:UnicodeBuffer;
```

Comment:
Why did you move this var declaration? The scope of vars should be kept as small as possible IMO.